rpc_jobs/bump_snapshots.yml

```
@@ -14,156 +14,165 @@
+                  def buildResult = build(
+                    job: 'PM_rpc-gating-master-snapshot-snapshot-test',
+                    wait: true,
+                    propagate: false,
+                    parameters: [
+                      [
+                        $class: "StringParameterValue",
+                        name: "RPC_GATING_BRANCH",
+                        value: env.RPC_GATING_BRANCH,
+                      ],
+                      [
+                        $class: "StringParameterValue",
+                        name: "BRANCH",
+                        value: env.RPC_GATING_BRANCH,
+                      ],
+                      [
+                        $class: "StringParameterValue",
+                        name: "IMAGE",
+                        value: "rpc-${release['latest']}-${release['image']}",
+                      ]
                     ]
-                  ]
-                )
-                if (buildResult.result != "SUCCESS") {
-                  println "Build failed, no updates will take place for image rpc-${release['latest']}-${release['image']}"
-                }
-                release['result'] = buildResult.result
-              } // parallelBuilds
-            } else {
-              println "rpc-${release['latest']}-${release['image']} has failed 7 times consecutively, creating RE issue for follow-up"
-              sendFailure = true
-            } // if
-          } // for
+                  )
+                  if (buildResult.result != "SUCCESS") {
+                    println "Build failed, no updates will take place for image rpc-${release['latest']}-${release['image']}"
+                  }
+                  release['result'] = buildResult.result
+                } // parallelBuilds
+              } else {
+                println "rpc-${release['latest']}-${release['image']} has failed 7 times consecutively, creating RE issue for follow-up"
+                sendFailure = true
+              } // if
+            } // for
 
-          parallel parallelBuilds
+            parallel parallelBuilds
 
-          for ( release in releases ) {
-            if (release['result'] == "SUCCESS") {
-              sh """#!/bin/bash -xe
-                sed -i "s/rpc-${release['previous']}-${release['image']}/rpc-${release['latest']}-${release['image']}/g" rpc_jobs/*.yml
-                sed -i "s/rpc-${release['previous']}-${release['image']}/rpc-${release['latest']}-${release['image']}/g" nodepool/files/nodepool.yml
-              """
-            }
-          } // for
+            for ( release in releases ) {
+              if (release['result'] == "SUCCESS") {
+                sh """#!/bin/bash -xe
+                  sed -i "s/rpc-${release['previous']}-${release['image']}/rpc-${release['latest']}-${release['image']}/g" rpc_jobs/*.yml
+                  sed -i "s/rpc-${release['previous']}-${release['image']}/rpc-${release['latest']}-${release['image']}/g" nodepool/files/nodepool.yml.j2
```

The directory has also changed.
```suggestion
                  sed -i "s/rpc-${release['previous']}-${release['image']}/rpc-${release['latest']}-${release['image']}/g" nodepool/templates/nodepool.yml.j2
```